.github/workflows/ci.yml

```diff
@@ -49,20 +49,11 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Prepare condarc file
-      run: |
-          echo "notify_outdated_conda: false" > .condarc
-
-    - uses: goanpeca/setup-miniconda@v1
-      with:
-        miniconda-version: 4.7.12
-        environment-file: environment.yml
-        activate-environment: vector
-        condarc-file: .condarc
+    - name: Install environment
+      run: conda create env
```

Comment:
```suggestion
      run: conda create env --file environment.yml --name vector
```

Reply:
This should have been the default behavior? I'll try it...

Reply:
My bad, I wrote this backwards! Sorry!